tools/make-rpms

```diff
@@ -50,22 +54,59 @@ while [ $# -gt 0 ]; do
     shift
 done
 
-if [ $# -gt 1 ]; then
+if [ $# -gt 1 -o $# = 1 -a -n "${build_in_place}" ]; then
     usage
     exit 2
 fi
 
-srpm="$($base/tools/make-srpm ${1-})"
-
 tmpdir=$(mktemp -d $PWD/rpm-build.XXXXXX)
+
+if [ -n "${build_in_place}" ]; then
+    test -f "${base}/configure" || NOCONFIGURE=1 "${base}/autogen.sh"
+    version="$(sed -n "s/^PACKAGE_VERSION='\(.*\)'\$/\1/p" "${base}/configure")"
+    test -n "${version}"
+
+    spec="${tmpdir}/cockpit.spec"
+    sed "s/^Version:.*$/Version: ${version}/" "${base}/tools/cockpit.spec" > "${spec}"
+    "${base}/tools/gen-spec-dependencies" "${spec}"
```

Comment:
Can we factor this to make_dist.py or even Makefile.am, instead of repeating it here and in make-srpm?

Reply:
This I agree with a bit more.  It might be nice to clean this up a bit.  In fact, I sort of had the idea that maybe we want to teach `gen-spec-dependencies` to also fill in the version information.  I can't fathom for the life of me why we ship a specific version of the tarball with a spec file containing version 0.